src/Iris/Env.hs

```diff
@@ -144,6 +152,15 @@ newtype CliEnvException = CliEnvException
 
 {- |
 
+@since 0.0.0.0
+-}
```

Comment:
No need to add `@since` annotation here because this data type is internal and not exported.

You can write some other documentation like `Wrapper around @cmd@ with additional predefined fields`